assets/src/components/parts/janus-navigation-button/NavigationButton.tsx

```diff
@@ -296,6 +310,11 @@ const NavigationButton: React.FC<PartComponentProps<NavButtonModel>> = (props) =
                 setButtonTitle(sTitle);
               }
 
+              const src = changes[`stage.${id}.src`];
```

Comment:
I'm not sure I really like "src" to be the property name, could it be `image-src` maybe instead?